<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_PointsLatticeOptions</class>
 <widget class="QWidget" name="LC_PointsLatticeOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="3">
    <widget class="QSpinBox" name="sbNumY">
     <property name="toolTip">
      <string>Number of points by lattice Y direction (count of lattice rows)</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lPointsY">
     <property name="text">
      <string>Rows Count:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lPointsX">
     <property name="text">
      <string>Columns Count:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbNumX">
     <property name="toolTip">
      <string>Number of points by X lattice direction (count of lattice columns)</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cbAdjustLastPoint">
     <property name="toolTip">
      <string>If selected, last point of lattice will be adjusted accoriding to position of first point, so lattice will be rect or quadrangle</string>
     </property>
     <property name="text">
      <string>Adjust last point to first</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
